pkg/volume/csi/csi_mounter_test.go

```diff
@@ -144,6 +144,12 @@ func TestMounterSetUp(t *testing.T) {
 	if err := csiMounter.SetUp(&fsGroup); err != nil {
 		t.Fatalf("mounter.Setup failed: %v", err)
 	}
+
+	//Test the default value of file system type is not overriden
+	if len(csiMounter.spec.PersistentVolume.Spec.CSI.FSType) != 0 {
+		t.Fatalf("default value of file system type was overriden by type %s" , csiMounter.spec.PersistentVolume.Spec.CSI.FSType)
```

Comment:
This could just be Errorf instead of Fatalf.  Fatalf will exit and not run the remainder of the test.